picard/ui/options/scripting.py

```diff
@@ -166,12 +166,17 @@ def __init__(self, parent):
 
         def process_html(html, function):
             if not html:
-                return ''
+                html = ''
+            template = '<dt>%s%s</dt><dd>%s</dd>'
+            if function.module is not None and function.module != 'picard.script':
+                module = ' [' + function.module + ']'
+            else:
+                module = ''
             try:
                 firstline, remaining = html.split("\n", 1)
-                return '<dt>' + firstline + '</dt><dd>' + remaining + '</dd>'
+                return template % (firstline, module, remaining)
             except ValueError:
-                return '<dt>$' + function.name + '(...)</dt><dd>' + html + '</dd>'
+                return template % ("$%s()" % function.name, module, html)
 
```

Comment:
A plugin script function can be documented as:

```python
@script_function(documentation="test")
def keep():
   code...
```
![image](https://user-images.githubusercontent.com/151042/76402599-ef9b0100-6383-11ea-85f0-10660e8012a9.png)

Reply:
Note, in the image above, there's also a translation test (it works!).